docs/implementation_progress.md

```diff
@@ -0,0 +1,145 @@
+## About
+
+When bootstrapping StableHLO from MHLO, we have inherited MHLO's implementation
+of verification. It already has significant coverage of the opset, but there's
+still plenty to do to review the existing implementations for completeness and
+provide new implementations where none exist.
+
+This live document is for the developers to track the progress on various
+aspects of the implementation - pretty printing, verification, type inference,
+specification, interpreter etc.
+
+### How to use it
+
+The progress of a StableHLO op, as mentioned in the corresponding row, on a
+particular implementation aspect, as mentioned in the corresponding column, is
+tracked using one of the following tracking labels.
+
+ - Generic labels
+    - **yes**: complete
+    - **wip**: semi-complete: Work in progress or under review.
+    - **no**: not complete yet, but plan to do so in near future.
+    - **no-plan**: not complete and not planned.
```

Comment:
Do we expect to use this label for something right away? If not, let's drop it. We can always bring it back if needed later.